examples/format-imperative/.expected/config.yaml

```diff
@@ -1,2 +1,4 @@
 testType: eval
 image: gcr.io/kpt-fn/format:unstable
+# Skip this until we figure out what we should do
+skip: true
```

Comment:
`kpt fn eval --image=format:unstable` doesn't produce any diff, while `kpt fn source foo | docker run -i gcr.io/kpt-fn/format:unstable | kpt fn sink new-dir` still works.

This PR is to fix the CI and unblock others. 
We need to figure out what we should do and unskip it in a followup PR.

Reply:
You can just delete the diff.patch instead of skipping. That means, format function doesn't change the order of the fields and so doesn't generate the diff.patch